.devcontainer/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM ghcr.io/astral-sh/uv:0.7 as uv_installer
+
+FROM mcr.microsoft.com/devcontainers/base:ubuntu-24.04 as graalvm_installer
+ARG GRAALVM_DOWNLOAD_URL=https://download.oracle.com/graalvm/23/latest/graalvm-jdk-23_linux-x64_bin.tar.gz
+ARG GRAALVM_INSTALL_DIR=/opt
+RUN curl -L ${GRAALVM_DOWNLOAD_URL} -o /tmp/graalvm.tar.gz && \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)